source/experimental/ConsoleApplication/ConsoleApplication.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27703.2042
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ConsoleApplication", "ConsoleApplication\ConsoleApplication.vcxproj", "{AA82D415-E4BA-48C7-85D8-6F675D70DED9}"
```

Comment:
>ConsoleApplication [](start = 53, length = 18)

Perhaps this should be renamed to a slightly less generic name.

Reply:
Renamed the console application `adapt`.